model/finished_stages.go

```diff
@@ -7,6 +7,8 @@ type Stage struct {
 	Workflows []Workflow `json:"workflows"`
 }
 
+type FinishedWorkflows []Workflow
```

Comment:
nit: We might want to rename the file to something more generic as `FinishedStages` and `FinishedWorkflows` are now at the same level.